### ISSUE TITLE
Store information within one account

### ISSUE THREAD

**Issue by karan-k-deepr** [brave/sync#294]
Accouding to Sync documentation on https://github.com/brave/sync/wiki/Design. It is written that brave will not able to recover bookmarks and other information if we remove brave from all devices.
It is the case with me where I am using brave on my laptop and using brave on my smartphone. But I do not want both devices to be in sync, because I am using a laptop for my official purpose and smartphone for my personal purpose. So, I do not want to mix my personal bookmarks with professional. So I can't recover my data if I format my laptop or smartphone. 

Can you please add this feature so that I can store my data using one account, either google account or brave own account. I prefer adding Google account for sync purpose.

**Comment by marnovo** [comment]
Ditto. This is one dealbreaker so far for me and most of my coworkers to move all-in to Brave. Feeling like some issue can make you lose all your browser data (nowadays a big deal) is a tad scary to make it your main workhorse.

**Comment by diracdeltas** [comment]
The exactly sentence is "if you delete Brave browser from all your devices, you will probably not be able to recover old browsing data." The **probably** is important, because there is a way to recover the data. Simply create a backup of the sync code words (the ones which you use to set up the 2nd device) and you're good to go.

i'll edit the wiki

**Comment by diracdeltas** [comment]
changed to `if you delete Brave browser from all your devices and don't have a backup of your sync encryption key, you will not be able to decrypt old browsing data.`

**Comment by karan-k-deepr** [comment]
> The exactly sentence is "if you delete Brave browser from all your devices, you will probably not be able to recover old browsing data." The **probably** is important, because there is a way to recover the data. Simply create a backup of the sync code words (the ones which you use to set up the 2nd device) and you're good to go.
> 
> i'll edit the wiki

That looks Promising, but can I create backup sync `codes` if I use only one device? Because there are many users like me who are using `Brave` on one system only, and do not require another device for sync, but want to restore previous backup data after the format of the device.

**Comment by diracdeltas** [comment]
> can I create backup sync `codes` if I use only one device

Yes, I believe there's a UI button to backup your codes without adding a device. If not then you can go through the 'add device' flow up to the point where it shows you the code then exit.

**Comment by srirambv** [comment]
>I believe there's a UI button to backup your codes without adding a device.  

Desktop requries two devices for chain to stick. Android can do it without adding a device

**Comment by diracdeltas** [comment]
I just confirmed on desktop that this is possible.

1. go to brave://sync/
2. click start new sync chain
3. click 'computer'
4. it shows the sync code words. this is your encryption key. 
5. click cancel
